driver/main.c

```diff
@@ -2349,6 +2366,7 @@ TRACEPOINT_PROBE(syscall_exit_probe, struct pt_regs *regs, long ret)
 #ifdef _HAS_SOCKETCALL
 	if (id == socketcall_syscall)
 	{
+		/// TODO: we cannot return 0 we need to handle this case in all drivers.
```

Comment:
Not sure what this comment means :(

Reply:
That if `convert_network_syscalls` fails, we need to handle that; right @Andreagit97 ?
FOr now, it is left unhandled in all 3 drivers; we will need to get back and fix that (in another PR of course)

Reply:
I think i found a quick and clean solution: 
```
if (id > 0)
		{
			event_data.is_socketcall = true;
		}
		else
		{
			// reset NR_socketcall and send a generic event
			id = socketcall_syscall;
		}
```
Then we will send a GENERIC{E,X} event with socketcall id.
WDYT? @Andreagit97 @gnosek

Reply:
I just pushed the changes. PTAL :) 
I think we are ok now, let me know!